scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -130,6 +130,11 @@ trait ClassLikeSupport:
       case _ => None
   )
 
+  private def parseInheritedMember(s: Tree): Option[Member] = processTreeOpt(s)(s match
+      case c: ClassDef if c.symbol.shouldDocumentClasslike && !c.symbol.isGiven => Some(parseClasslike(c, signatureOnly = true))
+      case other => parseMember(other)
```

Comment:
Can we keep the indentation uniform at 2 spaces? Mismatched indentation is a problem when diffing git changes. We'd like to avoid that now that our history is part of Dotty repo.

Reply:
fixed indentation